src/components/Sum.jsx

```diff
@@ -1 +1,14 @@
 // export function Sum() {}
+import React from 'react';
+
+export function Sum(props) {
+  const a = props.a || 0;
+  const b = props.b || 0;
```

Comment:
descructure the props ass you pass them to the component and give default values with `=`